<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WinAmp</string>
  </property>
  <widget class="QPushButton" name="pushButtonOpen">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPrev">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPlay">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPause">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonStop">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNext">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderVolume">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderTrack">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>441</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelVolume">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Volume:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDuration">
   <property name="geometry">
    <rect>
     <x>376</x>
     <y>80</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Duration:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPlayed">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Played:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelComposition">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Compozition:</string>
   </property>
  </widget>
  <widget class="QTableView" name="playListView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>521</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxLoop">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>30</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Loop</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxShuffle">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>80</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Shuffle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMute">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>130</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
